e2e-tests/gef/cypress/e2e/facility/review-ukef-approval/facility-end-date-disabled/review-ukef-approval-with-issued-and-unissued-facilities-MIA-and-resubmit-to-ukef.spec.js

```diff
@@ -491,9 +492,7 @@ context('Check activity feed', () => {
       applicationActivities.activityTimeline().contains('Bank facility stage changed');
 
       // contains submission message
-      applicationActivities
-        .activityTimeline()
-        .contains(`${CONSTANTS.PORTAL_ACTIVITY_LABEL.MIN_SUBMISSION} by ${BANK1_CHECKER1.firstname} ${BANK1_CHECKER1.surname}`);
+      applicationActivities.activityTimeline().contains(`${PORTAL_ACTIVITY_LABEL.MIN_SUBMISSION} by ${BANK1_CHECKER1.firstname} ${BANK1_CHECKER1.surname}`);
```

Comment:
would it be better if the string is stored in a const? same for below files if you agree

Reply:
I just changed the constant - really don't want to scope creep this PR.

There are a tonne of improvements that could be made in this E2E test.